ext_emconf.php

```diff
@@ -3,7 +3,7 @@
 /***************************************************************
  * Extension Manager/Repository config file for ext "core_upgrader".
  *
- * Auto generated 05-08-2020 13:47
+ * Auto generated 11-08-2020 15:35
```

Comment:
Please do not change the ext_emconf.php manually. This is auto generated on new release.